src/index.js

```diff
@@ -1,5 +1,8 @@
+import keyboardeventKeyPolyfill from 'keyboardevent-key-polyfill'
 import { deprecateComponent } from './utils/deprecate'
 
+keyboardeventKeyPolyfill.polyfill()
```

Comment:
Considering the other comment surrounding babel-polyfill, I could see pulling the functionality of this polyfill into a util instead so that we're not touching the global namespace.  This polyfill of course tampers with the [KeyboardEvent.prototype](https://github.com/cvan/keyboardevent-key-polyfill/blob/master/index.js#L109).